### DISCUSSION TITLE
using virtual foreign * across two databases

### DISCUSSION THREAD

**Discussion by wibrt** [dbeaver/dbeaver#8834] [Q&A] [answered]
#### System information: 
- DBeaver CE 7.0.5

#### Connection specification:
- For example 1 remote jdbc postgres, with tableP
- 1 local jdbc csv with tableC
So 2 connections to different types of datasources

#### Describe your question:
Can i write a query that for examples joins tableP with tableC?
Can i write a query that using tableC in a select that is part of a insert on tableP?

So basically: is this possible and how would you that?
Combining tables from different datasources?

I know you can use sql/med if the datasource has implemented it,
but as dbeaver has these virtual * concepts, i wonder if this is possible?

Reference example https://www.postgresql.eu/events/pgconfeu2019/sessions/session/2825/slides/213/FDW-DBeaver.pdf
* this is done with fdw (sql/med)
* but what about the vfk in the presentation?

**Comment by uslss** [comment, marked as answer]
Hi @wibrt ,
unfortunately, it is not possible to use JOIN across two databases using VFK.
You could use FDW or Presto database for it.
